### PR TITLE
Add missing spaces after commas in docs

### PR DESCRIPTION
**Pull request by anntzer**
... and various other small docstring fixes.

## PR Checklist

- [ ] Has Pytest style unit tests
- [ ] Code is [Flake 8](http://flake8.pycqa.org/en/latest/) compliant
- [ ] New features are documented, with examples if plot related
- [ ] Documentation is sphinx and numpydoc compliant
- [ ] Added an entry to doc/users/next_whats_new/ if major new feature (follow instructions in README.rst there)
- [ ] Documented in doc/api/api_changes.rst if API changed in a backward-incompatible way

### CODE REVIEW

File: lib/matplotlib/collections.py [hunk 1]
```diff
@@ -1149,15 +1149,15 @@ def __init__(self,
...
             facecolors = [cm.jet(x) for x in np.random.rand(20)]
-            black = (0,0,0,1)
+            black = (0, 0, 0, 1)
 
             collection = RegularPolyCollection(
                 numsides=5, # a pentagon
                 rotation=0, sizes=(50,),
                 facecolors=facecolors,
-                edgecolors=(black,),
+                edgecolors=("k",),
```
Comment (timhoffm) [inline lib/matplotlib/collections.py]:
```suggestion
                edgecolors=("black",),
```
Example code should be as readable as possible :smile:

File: examples/user_interfaces/embedding_in_wx4_sgskip.py [hunk 2]
```diff
@@ -21,29 +21,21 @@ class MyNavigationToolbar(NavigationToolbar):
...
-                color=rgb)
-        self.canvas.draw()
+        # add some text to the axes in a random location in axes coords with a
+        # random color
+        ax = self.canvas.figure.axes[0]  # get the axes
+        x, y = np.random.rand(2)  # generate a random location
+        rgb = np.random.rand(3)  # generate a random color
+        ax.text(  # add the text
+            x, y, 'You clicked me', transform=ax.transAxes, color=rgb)
+        self.canvas.draw()  # draw
```
Comment (timhoffm) [inline examples/user_interfaces/embedding_in_wx4_sgskip.py]:
```suggestion
        self.canvas.draw()
```
No information, so just leave out. The `# add the text` and `# get the axes` are almost as meaningless and could as well be removed.

File: lib/matplotlib/collections.py (hunk 1 above)
Comment (anntzer) [reply lib/matplotlib/collections.py]: fixed (and deleted the unused definition `black = (0, 0, 0, 1)` just above)

File: examples/user_interfaces/embedding_in_wx4_sgskip.py (hunk 2 above)
Comment (anntzer) [reply examples/user_interfaces/embedding_in_wx4_sgskip.py]: fair enough, removed all of them

**Comment by QuLogic** [comment]
Conflicts...

**Comment by anntzer** [comment]
rebased

**Comment by jklymak** [comment]
Hopefully this doesn't engender too many other rebases...

**Pull request merged**